website/docs/security/kyverno/baseline-pss.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Enforcing Pod Security Standards"
+sidebar_position: 134
+---
+
+As discussed in the introduction for [Pod Security Standards (PSS)](../pod-security-standards/) section, there are 3 pre-defined Policy levels, **Privileged**, **Baseline**, and **Restricted**. While it is recommended to setup a Restricted PSS, it can cause unintended behavior on the application level unless properly set. To get started it is recommended to setup a Baseline Policy that will prevent known Privileged escalations such as Containers accessing HostProcess, HostPath, HostPorts or allow traffic snooping for example, being possible to setup individual policies to restrict or disallow those privileged access to containers.
+
+A Kyverno Baseline Policy will help to restrict all the known privileged escalation under a single policy, and also maintain and update the Policy regularly adding the latest found vulnerabilities to the Policy.
+
+Privileged containers can do almost everything that the host can do and are often used in CI/CD pipelines to allow building and publishing Container images.
+With the now fixed [CVE-2022-23648](https://github.com/containerd/containerd/security/advisories/GHSA-crp2-qrr5-8pq7) any bad actor, could escape the privileged container by abusing the Control Groups `release_agent` functionality to execute arbitrary commands on the container host.
+
+In this lab, we will create a privileged Pod on our EKS cluster and check its privileged permissions. To do that, run the following command.
+
+```bash
+$ kubectl run -ti privileged-pod --image=centos --privileged=true --rm=true --restart=Never
```

Comment:
We avoid interactive shells inside pods. Is there another sequence of commands we can run? For example running the pod then using a non-interactive exec?

Reply:
Changed to this

```
kubectl run privileged-pod --image=centos --restart=Never --privileged --attach --rm --command -- /usr/sbin/capsh --print
```